docs/guides/admin/context-links.md

```diff
@@ -0,0 +1,139 @@
+# Context Links
+
+This feature was introduced as part of [#2439](https://github.com/google/timesketch/issues/2439) and allows linking of specific event attributes and values to external lookup services for an easy analyst workflow.
```

Comment:
Not sure if it is adding value to link to the PR that added the feature

Reply:
The link is pointing to the issue that tracked the implementation. However, I can see that it adds no real value so I removed it.